### DISCUSSION TITLE
[@mantine/hooks] - useRandomInterval

### DISCUSSION THREAD

**Discussion by echoghi** [mantinedev/mantine#3478] [Feature requests]
Would you guys be interested in adding a hook that produces a randomized interval within a given range? 

It's based on [Josh Comeau's hook](https://www.joshwcomeau.com/snippets/react-hooks/use-random-interval/) but refactored to work the same as @mantine/hooks/useInterval by returning an object with functions to start, stop, and toggle + a boolean determining if the interval is active. 

![useRandomInterval](https://user-images.githubusercontent.com/11097578/216460150-c4633433-40dc-4f0d-b58e-c0674de80db6.png)

**Comment by rtivital** [comment]
No, it is a very niche hook, that does not have many use cases.
